src/fidesops/ops/service/connectors/email_connector.py

```diff
@@ -28,9 +48,38 @@ def close(self) -> None:
 
     def test_connection(self) -> Optional[ConnectionTestStatus]:
         """
-        Override to skip connection test for now
+        Sends an email to the "test_email" configured, just to establish that the email workflow is working.
         """
-        return ConnectionTestStatus.skipped
+        config = EmailSchema(**self.configuration.secrets or {})
+        logger.info("Starting test connection to %s", self.configuration.key)
+
+        db = Session.object_session(self.configuration)
+
+        try:
+            dispatch_email(
+                db=db,
+                action_type=EmailActionType.EMAIL_ERASURE_REQUEST_FULFILLMENT,
+                to_email=config.test_email,
+                email_body_params={
+                    "test_collection": CheckpointActionRequired(
+                        step=CurrentStep.erasure,
+                        collection=CollectionAddress("test_dataset", "test_collection"),
+                        action_needed=[
+                            ManualAction(
+                                locators={"id": ["example_id"]},
+                                get=None,
+                                update={
+                                    "test_field": "null_rewrite",
+                                },
+                            )
+                        ],
+                    )
+                },
+            )
+        except EmailDispatchException as exc:
+            logger.info("Email connector test failed with exception %s", Pii(exc))
+            return ConnectionTestStatus.failed
+        return ConnectionTestStatus.succeeded
```

Comment:
If a test_email is configured (you'd set up a test_email that you had access to), you can use this to send a test email to yourself.  The data inside is just dummy data. It really just confirms your email config is working.